### PR TITLE
APP-94627 pendo web (actions) doc

### PR DESCRIPTION
**Pull request by wordythebyrd**
### Proposed changes

New upgrade documentation for the Pendo Web (Actions) Destination.

### Merge timing
ASAP once approved

### CODE REVIEW

**Comment by wordythebyrd** [comment]
Hey @joe-ayoub-segment, here's the documentation update we talked about on Monday.

File: src/connections/destinations/catalog/pendo-web-actions/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+---
+title: Pendo Web (Actions) Destination
```
Comment (forstisabella) [inline src/connections/destinations/catalog/pendo-web-actions/index.md:2]: For context: the `id` is generated by Segment's Public API - including it here autogenerates info like the actions-fields, and adds this page to the [destination catalog page](https://segment.com/docs/connections/destinations/catalog/). The `hidden` and `beta` attributes denote that this is in private beta. 

```suggestion
title: Pendo Web (Actions) Destination
id: 6501a4325a8a629197cdd691
hidden: true
beta: true
```

File: src/connections/destinations/catalog/pendo-web-actions/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+{% include content/ajs-upgrade.md %}
+
+<!-- In the section below, explain the value of this actions-based destination over the classic version, if applicable. If you don't have a classic version of the destination, remove this section. -->
+
+## Benefits of Pendo Web (Actions) vs Pendo Classic
+
+Pendo Web (Actions) provides the following benefits over the classic Pendo destination:
+
+- **Support for `track`, `identify`, and `group`**. The classic Pendo destination did not support `track`, and required an additional webhook destination be configured.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/pendo-web-actions/index.md:32]:
```suggestion
- **Support for Track, Identify, and Group calls**. The classic Pendo destination did not support Track calls and required users to configure an additional Webhook destination.
```

File: src/connections/destinations/catalog/pendo-web-actions/index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+- **Full region support**. Setup allows the destination to be configured for US, EU, US restricted, or Japan.
+- **Supports CNAME for Pendo**.  Works with subscriptions using Pendo's custom CNAME feature.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/pendo-web-actions/index.md:34]:
```suggestion
- **Supports CNAME for Pendo**. Works with subscriptions using Pendo's custom CNAME feature.
```

File: src/connections/destinations/catalog/pendo-web-actions/index.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+Additional Context
+
+Include additional information that you think will be useful to the user here. For information that is specific to an individual mapping, please add that as a comment so that the Segment docs team can include it in the auto-generated content for that mapping.
+-->
+
+<!-- If applicable, add information regarding the migration from a classic destination to an Actions-based version below -->
+
+## Migration from the classic Pendo destination
+
+Remove the classic Pendo destination and webhook destination from your Segment account before adding the Pendo Web (Actions) destination.
```
Comment (forstisabella) [inline src/connections/destinations/catalog/pendo-web-actions/index.md:62]:
```suggestion
Remove the classic Pendo destination and webhook destination from your Segment workspace before adding the Pendo Web (Actions) destination.
```

**Review by forstisabella — APPROVED** [review]
🚀

**Comment by wordythebyrd** [comment]
Thanks @forstisabella

**Review by tcgilbert — CHANGES_REQUESTED** [review]
@wordythebyrd could you please change the name of the folder to `actions-pendo-web`

- current: `pendo-web-actions`
- change to: `actions-pendo-web`

**Comment by wordythebyrd** [comment]
> @wordythebyrd could you please change the name of the folder to `actions-pendo-web`
> 
> * current: `pendo-web-actions`
> * change to: `actions-pendo-web`

I've updated the folder name, thank you.

**Review by tcgilbert — APPROVED** [review]

**Pull request merged**